### PR TITLE
s3-sink sns-source docs changed

### PR DESCRIPTION
**Pull request by SLyyq10**
Signed-off-by: sling <18102030131@163.com>
A new pull request for these two docs because previous pull requests disappeared.
Docs both changed.

### CODE REVIEW

File: user-manual/connectors/sink/s3.md [hunk 1]
```diff
@@ -0,0 +1,207 @@
...
+}
+```
+| Secrets            | Description                                                          | Example                       |Required|
+|:-------------------|:---------------------------------------------------------------------|:------------------------------|:-------|
+| awsAccessKeyID     | `awsAccessKeyID` is the Access key ID of your aws credential.        | "BASE64VALUEOFYOURACCESSKEY=" |**YES** |
+| awsSecretAccessKey | `awsSecretAccessKey` is the Secret access key of youraws credential. | "BASE64VALUEOFYOURSECRETKEY=" |**YES** |
+
+### Run the Amazon S3 Sink with Docker
+
+At first, you can **pull the image** of S3 Sink from dockerhub like following command.
```
Comment (Michaelg22) [inline user-manual/connectors/sink/s3.md:114]:
```suggestion
You can get the s3 Sink image from docker with the following command.
```

File: user-manual/connectors/source/sns.md [hunk 2]
```diff
@@ -0,0 +1,265 @@
...
+| Secrets            | Description                                                          | Example                       |Required|
+|:-------------------|:---------------------------------------------------------------------|:------------------------------|:-------|
+| awsAccessKeyID     | `awsAccessKeyID` is the Access key ID of your aws credential.        | "BASE64VALUEOFYOURACCESSKEY=" |**YES** |
+| awsSecretAccessKey | `awsSecretAccessKey` is the Secret access key of youraws credential. | "BASE64VALUEOFYOURSECRETKEY=" |**YES** |
+
+### Run the Display Connector to Print Events
+
+Before launching an Amazon SNS Source, we create a Display Connector to receive and **print CloudEvents** sent from the SNS Source.
+
+You can use following command to pull a Display Connector's image from duckerhub.
```
Comment (Michaelg22) [inline user-manual/connectors/source/sns.md:176]:
```suggestion
You can use the following command to get the Display sink image.
```

File: user-manual/connectors/source/sns.md [hunk 3]
```diff
@@ -0,0 +1,265 @@
...
+You can use [ngrok] to **expose** your local port to the **public network**. After you register, login and install ngrok, 
+you can use the following command to get a address with specified port accessible to the public network.
+
+```shell 
+[user@localhost]$ ngrok http 8082
+``` 
+
+### Run the Amazon SNS Source with Docker
+
+You can use following command to pull the SNS Source's image.
```
Comment (Michaelg22) [inline user-manual/connectors/source/sns.md:200]:
```suggestion
You can get the SNS Source by using the following command.
```

**Review by Michaelg22 — CHANGES_REQUESTED** [review]

File: user-manual/connectors/source/sns.md [hunk 4]
```diff
@@ -0,0 +1,265 @@
...
+You can get the SNS Source by using the following command.
+
+```shell 
+docker pull vancehub/source-aws-sns:latest
+```
+
+Create your `config.json` and `secret.json`, and mount them to specific paths to run the SNS source using following command.
+
+In order to send events to Display Connector, set the `v_target` value as `http://host.docker.internal:8081` in your `config.json` file.
+The `host.docker.internal` indicates docker's host ip. You can also use the ip address of **container** where Display Connector runs. Try following
```
Comment (VedaGao) [inline user-manual/connectors/source/sns.md:209]:
```suggestion
The `host.docker.internal` indicates Docker's host IP. You can also use the IP address of **container** where Display Connector runs. Try following 
```

File: user-manual/connectors/source/sns.md [hunk 5]
```diff
@@ -0,0 +1,265 @@
...
+
+```shell 
+docker pull vancehub/source-aws-sns:latest
+```
+
+Create your `config.json` and `secret.json`, and mount them to specific paths to run the SNS source using following command.
+
+In order to send events to Display Connector, set the `v_target` value as `http://host.docker.internal:8081` in your `config.json` file.
+The `host.docker.internal` indicates docker's host ip. You can also use the ip address of **container** where Display Connector runs. Try following 
+command to get the ip address of container.
```
Comment (VedaGao) [inline user-manual/connectors/source/sns.md:210]:
```suggestion
command to get the IP address of container.
```

File: user-manual/connectors/source/sns.md [hunk 6]
```diff
@@ -0,0 +1,265 @@
...
+Use `docker ps` to get the Container id of Display Connector.
```
Comment (VedaGao) [inline user-manual/connectors/source/sns.md:211]:
```suggestion
Use `docker ps` to get the Container ID of Display Connector.
```

**Review by VedaGao — CHANGES_REQUESTED** [review]

File: user-manual/connectors/sink/s3.md [hunk 7]
```diff
@@ -0,0 +1,207 @@
...
+
+
+After S3 Sink Connector, you can see following logs:
+
+```shell 
+[root@iZ8vbhcwtixrzhsn023sghZ ~]# curl -X POST \
+> -d '{"specversion":"0.3","id":"b25e2717-a470-45a0-8231-985a99aa9416","type":"com.github.pull.create","source":"https://github.com/cloudevents/spec/pull/123","time":"2019-07-04T17:31:00.000Z","datacontenttype":"application/json","data":{"much":"wow"}}' \
+> -H'Content-Type:application/cloudevents+json' \
+> http://8.142.113.12:8080
+receive CloudEvent success
```
[the remark this answers is not in the document]
Comment (SLyyq10) [reply user-manual/connectors/sink/s3.md:148]: The message is a response replied by S3 Sink and it doesn't have 's'.

File: user-manual/connectors/sink/s3.md [hunk 8]
```diff
@@ -0,0 +1,207 @@
...
+
+### Run the Amazon S3 Sink with Docker
+
+You can get the S3 Sink image from Docker with the following command.
+
+```shell
+docker pull vancehub/sink-aws-s3:latest
+```
+
+Create your `config.json` and `secret.json`, and mount them to specific paths to run the S3 Sink using following command.
```
Comment (Michaelg22) [inline user-manual/connectors/sink/s3.md:120]:
```suggestion
After creating your 'config.json' and 'secret.json'  files, you can run the S3 Sink and mount the files with the following command.
```

File: user-manual/connectors/sink/s3.md [hunk 9]
```diff
@@ -0,0 +1,207 @@
...
+
+```shell
+docker run -v $(pwd)/secret.json:/vance/secret/secret.json -v $(pwd)/config.json:/vance/config/config.json -p 8080:8080 --rm vancehub/sink-aws-s3
+```
+
+### Verify the Amazon S3 Sink
+
+To verify the S3 Sink, you should send CloudEvents to address of S3 Sink. Try to use following `curl` command.
```
Comment (Michaelg22) [inline user-manual/connectors/sink/s3.md:128]:
```suggestion
To verify the S3 Sink, you should send CloudEvents to the address of S3 Sink. Try to use the  following `curl` command. 
```

File: user-manual/connectors/source/sns.md [hunk 10]
```diff
@@ -0,0 +1,265 @@
...
+
+### Run the Amazon SNS Source with Docker
+
+You can get the SNS Source by using the following command.
+
+```shell 
+docker pull vancehub/source-aws-sns:latest
+```
+
+Create your `config.json` and `secret.json`, and mount them to specific paths to run the SNS source using following command.
```
Comment (Michaelg22) [inline user-manual/connectors/source/sns.md:206]:
```suggestion
After creating your 'config.json' and 'secret.json'  files, you can run the SNS source and mount the files with the following command.
```

File: user-manual/connectors/source/sns.md [hunk 11]
```diff
@@ -0,0 +1,265 @@
...
+```
+
+```shell
+docker run -v $(pwd)/secret.json:/vance/secret/secret.json -v $(pwd)/config.json:/vance/config/config.json -p 8082:8082 --rm vancehub/source-aws-sns
+```
+
+### Verify the Amazon SNS Source
+
+You can verify if the Amazon SNS Source works properly by publishing a message to the SNS topic. To publish a message, 
+you should log in to the [AWS Management Console][aws-console] at first.Then visit the Simple Notification Service.
```
Comment (Michaelg22) [inline user-manual/connectors/source/sns.md:227]:
```suggestion
you should log in to the [AWS Management Console][aws-console] at first. Then visit the Simple Notification Service.
```

File: user-manual/connectors/source/sns.md [hunk 12]
```diff
@@ -0,0 +1,265 @@
...
+Choose the SNS Topics, and then you can create topics or publish messages to topics.
```
Comment (Michaelg22) [inline user-manual/connectors/source/sns.md:228]:
```suggestion
Choose the SNS Topics, and then you can create topics or publish messages to the topics.
```

**Review by Michaelg22 — CHANGES_REQUESTED** [review]

**Review by Michaelg22 — APPROVED** [review]

**Comment by Michaelg22** [comment]
Made a new pr with the new structure of these two documents

**Pull request closed without merging**